.babelrc

```diff
@@ -0,0 +1,3 @@
+{
+  "presets": ["react", "es2015", "stage-0"]
```

Comment:
Yeah, this was a copy-pasta.  But I do like generators, and the ::this.functionToBindToThis syntax.